Procfile

```diff
@@ -1,2 +1,3 @@
 web: bin/puma -C config/puma.rb
 worker: bin/sidekiq -C config/sidekiq.yml
+process_output_documents: rake output_documents:process
```

Comment:
`rake` or `bin/rake` ?

`bin/puma` and `bin/sidekiq` are Bundler shims whereas `bin/rake` [loads Spring](https://github.com/guidance-guarantee-programme/output/blob/master/bin/rake#L3)